src/EndpointStandalone.sol

```diff
@@ -4,28 +4,54 @@ pragma solidity >=0.6.12 <0.9.0;
 import "./Endpoint.sol";
 import "./interfaces/IManagerStandalone.sol";
 import "./interfaces/IEndpointStandalone.sol";
+import "./libraries/Implementation.sol";
+import "./libraries/external/ReentrancyGuardUpgradeable.sol";
 
-abstract contract EndpointStandalone is IEndpointStandalone, Endpoint {
-    /// updating bridgeManager requires a new Endpoint deployment.
+abstract contract EndpointStandalone is
+    IEndpointStandalone,
+    Endpoint,
+    Implementation,
+    ReentrancyGuardUpgradeable
+{
+    bytes32 public constant ENDPOINT_MANAGER_SLOT =
+        bytes32(uint256(keccak256("endpoint.manager")) - 1);
```

Comment:
Can you make this `ntt.endpointManager` instead please to stick with the ntt theme